concurrency/MpmcQueue.h

```diff
@@ -249,13 +282,21 @@ class MpmcQueue
 
     // Layout below must be maintained for performance. The non atomic member variables may not be place before read
     // cursor.
-    std::atomic<VersionedIndex> _readCursor;
-    uint32_t _cacheLineSeparator1[14];
-    std::atomic<VersionedIndex> _writeCursor;
-    uint32_t _cacheLineSeparator2[14];
-    const uint32_t _entriesPerCacheline;
-    const uint32_t _capacity;
-    const size_t _blockSize;
+    alignas(64) std::atomic<VersionedIndex> _readCursor;
```

Comment:
Replace the explicit line sepatator fields and forced the alignment on 64 bits. This is a bit more resilient as before we couldn't have sure if `readCursor` was not sharing the cache line with another atomic members in a different object. `alignas(64)` force it to be the first address on the cache line